src/serac/infrastructure/accelerator.hpp

```diff
@@ -145,6 +147,19 @@ using GPUArray = ExecArray<T, dim, ExecutionSpace::GPU>;
 template <typename T, int dim = 1>
 using UnifiedArray = ExecArray<T, dim, ExecutionSpace::Dynamic>;
 
+#else
+// If not a CUDA build then force all arrays to be CPU
```

Comment:
There may be a better way of handling this but it is late and I have dreams of disney.